docs/gno-infrastructure/validators/setting-up-a-new-chain.md

```diff
@@ -93,7 +93,7 @@ Let's break down the most important default settings:
 :::info Resetting the chain
 
 As mentioned, the working directory for the node is located in `data-dir`. To reset the chain, you need
-to delete this directory and start the node up again. If you are using the default node configuration, you can run
+to delete this directory and `genesis.json`, then start the node up again. If you are using the default node configuration, you can run
```

Comment:
@zivkovicmilos should we specify maybe that genesis.json should mostly be removed if doing local development? (ie. not on-chain?)